corehq/apps/style/templates/style/includes/global_navigation_bar.html

```diff
@@ -56,6 +56,15 @@
                     <i class="fa fa-dashboard icon-dashboard dropdown-icon"></i> {% trans "Current Subscription" %}
                 </a>
             </li>
+            {% if request.couch_user.is_superuser %}
+            <li class="nav-divider divider"></li>
+            <li class="dropdown-header nav-header">{% trans 'Superuser Options' %}</li>
+            <li>
+                <a href="{% url "raw_doc" %}?id={{ request.project|get_id }}" id="menuitem-view-domain-doc">
```

Comment:
nice. I didn't know about that https://github.com/dimagi/commcare-hq/pull/13565/commits/c443724ac990276017327de762b92598a6013f2f